src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1656,29 +1659,33 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 
```

Comment:
Unrelated, but the timesync stream in the vanilla EXTVISION is redundant because of the fall through.

Reply:
Indeed, removed